Source/Redemption/Private/DeadBody.cpp

```diff
@@ -0,0 +1,33 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "DeadBody.h"
+#include "InteractWidgetComponent.h"
+#include "Components/BoxComponent.h"
+
+// Sets default values
+ADeadBody::ADeadBody()
+{
+ 	// Set this actor to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
+	PrimaryActorTick.bCanEverTick = false;
+
+	Mesh = CreateDefaultSubobject<USkeletalMeshComponent>(TEXT("Body Mesh"));
+
+	InteractComp = CreateDefaultSubobject<UInteractWidgetComponent>(TEXT("Interact Comp"));
+	InteractComp->GetBoxComp()->SetupAttachment(GetRootComponent());
+}
+
+// Called when the game starts or when spawned
+void ADeadBody::BeginPlay()
+{
+	Super::BeginPlay();
+	
+}
+
+// Called every frame
+void ADeadBody::Tick(float DeltaTime)
```

Comment:
this function I believe will never be called since PrimaryActorTick.bCanEverTick is set to false. So I think you can just delete it?

Reply:
yes, but I want to make sure it will not be used before I delete it. Because if I need to use it later I will have to create it again. So I plan to delete it after the system that uses the dead bodies is complete. but if you want I can delete it now.